third_party/googletest/src/googletest/src/gtest.cc

```diff
@@ -3601,30 +3638,31 @@ void PrettyUnitTestResultPrinter::PrintSkippedTests(const UnitTest& unit_test) {
         continue;
       }
       ColoredPrintf(GTestColor::kGreen, "[  SKIPPED ] ");
-      printf("%s.%s", test_suite.name(), test_info.name());
-      printf("\n");
+      posix::PrintF("%s.%s", test_suite.name(), test_info.name());
+      posix::PrintF("\n");
     }
   }
 }
 
+#if !GTEST_OS_STARBOARD
 void PrettyUnitTestResultPrinter::OnTestIterationEnd(const UnitTest& unit_test,
                                                      int /*iteration*/) {
   ColoredPrintf(GTestColor::kGreen, "[==========] ");
-  printf("%s from %s ran.",
+  posix::PrintF("%s from %s ran.",
```

Comment:
Those are in `#if !GTEST_OS_SSTARBOARD` block, so shouldn't be modified here and below